cmd/manager/main.go

```diff
@@ -303,3 +314,46 @@ func initCustomZapLogger(level, encoding string) (*zap.Logger, error) {
 	}
 	return cfg.Build()
 }
+
+func configureDeletionProtectionFlags(config *Config) {
+	if config == nil {
+		return
+	}
+
+	objectDeletionSet := false
+	subObjectDeletionSet := false
+
+	flag.Visit(func(f *flag.Flag) {
+		if f.Name == "object-deletion-protection" {
```

Comment:
Consider making names of the flags `const`. I see this is also used in the flag parsing method